app/Notifications/NewRequestFillApprove.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
```

Comment:
Should probably not notify the filler if they're the approver.

Reply:
Added self check.